src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -15,22 +14,30 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 import wolox.training.exceptions.UserNotFoundException;
-import wolox.training.models.Users;
+import wolox.training.models.Book;
+import wolox.training.models.User;
+import wolox.training.repositories.IBookRepository;
 import wolox.training.repositories.IUserRepository;
 
 @RestController
-@RequestMapping("/api/users")
+@RequestMapping("/users")
 public class UserController {
 
-    @Autowired
-    private IUserRepository userRepository;
+    private final IUserRepository userRepository;
+    private final IBookRepository bookRepository;
+
+
+    public UserController(IUserRepository userRepository, IBookRepository bookRepository) {
+        this.userRepository = userRepository;
+        this.bookRepository = bookRepository;
+    }
 
     /**
      * @return all the users contained in the model
      */
-    @GetMapping("/all")
-    public Optional<List<Users>> findAll() {
-        List<Users> allUsers = new ArrayList<>();
+    @GetMapping()
+    public Optional<List<User>> findAll() {
+        List<User> allUsers = new ArrayList<>();
         if (userRepository.findAll().iterator().hasNext()) {
             userRepository.findAll().forEach(b -> allUsers.add(b));
             return Optional.of(allUsers);
```

Comment:
You can directly return the result from the repository interface. It returns an empty list when there are no users in the database.
```suggestion
    public List<User> findAll() {
        return userRepository.findAll();
```